heat/core/operations.py

```diff
@@ -385,18 +385,25 @@ def __reduce_op(x, partial_op, reduction_op, **kwargs):
 
     # no further checking needed, sanitize axis will raise the proper exceptions
     axis = stride_tricks.sanitize_axis(x.shape, kwargs.get("axis"))
-    split = x.split
+    if isinstance(axis, int):
+        axis = (axis,)
     keepdim = kwargs.get("keepdim")
+    split = x.split
 
+    # if local tensor is empty, replace it with the identity element
+    if 0 in x.lshape and (axis is None or (x.split in axis)):
+        neutral = kwargs.get("neutral")
```

Comment:
I would suggest to add `neutral` to the named arguments instead of passing it through the `kwargs`. This way we could also provide a description how this argument might be used.

Reply:
My reasoning behind that decision was that neutral elements are specific to their functions and I'm not sure the user should be tempted to specify a different value. 

What I still haven't done is implement the `initial` argument (see Issue #101) which goes more in the direction of what you're saying, I think. It's on my list 😬

Reply:
can you make an issue for this? (and tag it with the 'first issue' tag)

Reply:
@coquelin77 done, see #448.